src/app/interfaces/dynamic-form-schema.interface.ts

```diff
@@ -0,0 +1,35 @@
+import { FormArray } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { ChartSchemaNode } from 'app/interfaces/chart-release.interface';
+import { Option } from 'app/interfaces/option.interface';
+import { TreeNodeProvider } from 'app/modules/ix-forms/components/ix-explorer/ix-explorer.component';
+
+export interface DynamicFormSchema {
+  name: string;
+  description: string;
+  schema: DynamicFormSchemaNode[];
+}
+
+export interface DynamicFormSchemaNode {
+  variable: string;
+  type: string;
+  attrs?: DynamicFormSchemaNode[];
+  items?: DynamicFormSchemaNode[];
+  items_schema?: ChartSchemaNode[];
```

Comment:
ChartSchemaNode doesn't sound like something that belongs to global DynamicFormSchemaNode. If there is a mismatch between schemas used in acme-dns, charts and frontend code, we can have separate interfaces with some utility functions to translate backend-schemas to frontend schemas.